website/docs/r/service_networking_alb.html.markdown

```diff
@@ -0,0 +1,62 @@
+---
+subcategory: "Service Networking"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_alb"
+description: |-
+  Manages an Application Load Balancer.
+---
+
+# azurerm_alb
+
+Manages an Application Load Balancer.
```

Comment:
I'd really like to have some reference or information block regarding that this resource is part of Application Gateway for Containers.

I understand the naming dilemma this resource brings along, but the first use case (Application Gateway for Containers) should at least be mentioned in the documentation I'd say 😄
```suggestion
Manages an Application Load Balancer. Application Load Balancer is part of Application Gateway for Containers.
```